scripts/collect_steps.py

```diff
@@ -54,8 +54,21 @@
     "ipa::protocol::attribution::aggregate_credit::Step::compute_equality_checks",
     "ipa::protocol::attribution::aggregate_credit::Step::check_times_credit",
 ]
+# another one for modulus conversion
+MODULUS_CONVERSION_DYNAMIC_STEPS = [
+    "ipa::protocol::sort::SortStep::convert",
+    "ipa::protocol::attribution::AttributionStep::convert_helper_bits",
+    "ipa::protocol::boolean::solved_bits::Step::random_bits",
+    "ipa::protocol::attribution::aggregate_credit::Step::mod_conv_breakdown_key_bits",
+]
 MAXIMUM_DEPTH = 32
 MAXIMUM_BIT_LENGTH = 32
+MAXIMUM_CONVERT_BIT_LENGTH = 64
+
+# REPEAT_STEPS = {'ipa::protocol::step::BitOpStep::bit': 32,
+#                 'ipa::protocol::attribution::InteractionPatternStep::depth': 32,
+#                 'ipa::protocol::modulus_conversion::convert_shares::Step::mc': 64,
+#                 }
```

Comment:
```suggestion

```

Reply:
Oops.  This was me realizing that the design for this script was not good, then realizing that I didn't have time to fix it...